### DISCUSSION TITLE
How do I add a text translation？

### DISCUSSION THREAD

**Discussion by whiteMu** [jgraph/drawio#4523] [Q&A] [answered]
I want to change android to 安卓，I modified the dia_zh.txt file, but it doesn't work![image](https://github.com/jgraph/drawio/assets/17615371/b866bc73-ba6f-41a7-aa8b-bf06222b24c4)
![image](https://github.com/jgraph/drawio/assets/17615371/b0813840-bd45-4be2-b794-be8e81019956)

**Comment by davidjgraph** [comment]
It does work, I suspect you're not loading the file you think you're loading.

<img width="724" alt="Capture d’écran 2024-07-17 à 18 11 58" src="https://github.com/user-attachments/assets/5ade6b81-84c8-4b9b-b3c2-e02cd16eabaa">

**Reply by whiteMu** [reply, marked as answer]
> It does work, I suspect you're not loading the file you think you're loading.
> 
> <img alt="Capture d’écran 2024-07-17 à 18 11 58" width="724" src="https://private-user-images.githubusercontent.com/1761087/349604276-5ade6b81-84c8-4b9b-b3c2-e02cd16eabaa.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjEyNjU1MDEsIm5iZiI6MTcyMTI2NTIwMSwicGF0aCI6Ii8xNzYxMDg3LzM0OTYwNDI3Ni01YWRlNmI4MS04NGM4LTRiOWItYjNjMi1lMDJjZDE2ZWFiYWEucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDcxOCUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDA3MThUMDExMzIxWiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9ZGQxZjc3Y2ExYzEwMWFlMTFiYTQwYTQwYzMyM2U2YWU3MjM3ZDJmMjE1YWViOTEyYTExMDU4OWM4ZDlkYTg0MCZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.6SALlZriNZkT_EXDHCPobeku7tAWfpU-4g_izpt1KV4">
Thank you very much for replying to this question,I found out why it didn't work.Because the service worker.
![image](https://github.com/user-attachments/assets/4891aee6-9272-4929-80fa-aba314024ca1)
![image](https://github.com/user-attachments/assets/5a540b51-862d-4a73-9cf1-12ffb0659945)

**Comment by whiteMu** [comment]
clear side data，and It does work.
